pyiron_atomistics/project.py

```diff
@@ -511,9 +511,16 @@ def _calculation_validation(
     def inspect_periodic_table():
         return PeriodicTable()
 
+ @staticmethod
```

Comment:
Indentation is off, but in the web displayer it's hard for me to tell exactly how much more whitespace is needed